packages/office-ui-fabric-react/src/components/Rating/Rating.scss

```diff
@@ -8,111 +8,58 @@ $Rating-disabledColor: $ms-color-neutralTertiaryAlt;
 $Rating-smallIconSize: 16px;
 $Rating-largeIconSize: 20px;
 
-// Hide input field
-.input {
-  position: absolute;
-  opacity: 0;
-  top: 0px;
-}
-
-.container {
-  position: relative;
+.ratingStar {
   display: inline-block;
+  position: relative;
+}
 
-  &:hover {
-    .star:not(.starIsDisabled) {
-      color: $Rating-selectedColor;
-
-      @include high-contrast {
-        color: HighlightText;
-        background-color: Highlight;
-        -ms-high-contrast-adjust: none;
-      }
-
-      &.star:hover {
-        color: $Rating-hoverColor;
-
-        @include high-contrast {
-          color: HighlightText;
-        }
-
-        & ~ .star {
-          // Set default color for any star after hovered one
-          color: $Rating-defaultColor;
-
-          @include high-contrast {
-            color: GrayText;
-            background-color: Window;
-          }
-        }
-      }
-    }
-  }
-
-  .star {
-    display: inline-block;
-
-    text-align: center;
-
-    color: $Rating-defaultColor;
-
-    @include high-contrast {
-      color: WindowText;
-      -ms-high-contrast-adjust: none;
-    }
-
-    &.starIsSelected {
-      color: $Rating-selectedColor;
-
-      @include high-contrast {
-        color: HighlightText;
-        background-color: Highlight;
-        -ms-high-contrast-adjust: none;
-      }
-    }
+.ratingStarBack {
+  color: $Rating-defaultColor;
+  width: 100%;
 
-    &.starIsDisabled {
+  &.ratingStarDisabled {
+    color: $Rating-disabledColor;
+     @media screen and (-ms-high-contrast: active) {
       color: $Rating-disabledColor;
-
-      @include high-contrast {
-        color: GrayText;
-        background-color: Window;
-      }
-
-      .label {
-        cursor: default;
-      }
     }
+  }
+}
 
-    .label {
-      display: inline-block;
-      cursor: pointer;
-
-      font-size: $Rating-smallIconSize;
-      padding: 12px 0px;
+.ratingStarFront {
+  position: absolute;
+  height: 100%;
+  left: 0;
+  top: 0;
+  text-align: center;
+  vertical-align: middle;
+  overflow: hidden;
+  color: $Rating-selectedColor;
+   @media screen and (-ms-high-contrast: active) {
+      color: rgb(255, 255, 0);
```

Comment:
It would be preferable to use a system color here  instead of yellow here. As is, this is not very readable in black-on-white high contrast mode and it won't support customized contrast palettes.

Semantically, Highlight or HighlightText probably convey the meaning of "selected" here.

Reply:
updated with highlight